cluster-provision/k8s/1.29/manifests/cni.diff

```diff
@@ -1,90 +1,88 @@
---- a/cluster-provision/k8s/1.19/manifests/cni.do-not-change.yaml
-+++ b/cluster-provision/k8s/1.19/manifests/cni.do-not-change.yaml
-@@ -32,7 +32,12 @@ data:
+--- a/cluster-provision/k8s/1.28/manifests/cni.do-not-change.yaml
++++ b/cluster-provision/k8s/1.28/manifests/cni.do-not-change.yaml
```

Comment:
Should this be 1.29 ?

Reply:
Actually this shouldn't matter as the command `patch $cni_manifest $cni_diff` is smart enough to ignore the path above but just use the path arguments passed to patch command. Previously also 1.19 is there at some diff files and 1.24 in other diff files, but it was working fine.

Still let me change this to 1.29 (than 1.28) as that gives us an idea from which minimum version we bumped calico.

Reply:
Done